working-open.md

```diff
@@ -11,6 +11,7 @@ Hypha operates with democratic organizing principles informed by our [values](./
 # How and what?
 
 - We currently accommodate redactions and going off-the-record for meeting notes. Participants can request to stop taking notes, or redact and correct notes after they have been taken.
+- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and partners. An employee or contractor with access to such information is responsible for keeping this information confidential.
```

Comment:
```suggestion
- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and partners. Anyone with access to confidential information is responsible for doing their best to keep confidential information secret and expected to adhere to these guidelines.
```
Mirroring language from our independent contractor agreement and IP assignment.

Also I think you may want to use the concept/explain what PII is here if you mention it below.

Reply:
What is personal information?

Under PIPEDA, personal information includes any factual or subjective information, recorded or not, about an identifiable individual. This includes information in any form, such as:

    age, name, ID numbers, income, ethnic origin, or blood type;
    opinions, evaluations, comments, social status, or disciplinary actions; and
    employee files, credit records, loan records, medical records, existence of a dispute between a consumer and a merchant, intentions (for example, to acquire goods or services, or change jobs).

https://www.priv.gc.ca/en/privacy-topics/privacy-laws-in-canada/the-personal-information-protection-and-electronic-documents-act-pipeda/pipeda_brief/#_h2

Reply:
```suggestion
- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and collaborators. Anyone with access to confidential information is responsible for doing their best to keep that information secret and are expected to adhere to these guidelines.
```